lib/index.ts

```diff
@@ -201,13 +197,7 @@ class ProgressReporter {
 			try {
 				let downloadedPercentage;
 				let extractedPercentage;
-				({ id, status } = evt);
-				if (id == null) {
-					id = '';
-				}
-				if (status == null) {
-					status = '';
-				}
+				({ id = '', status = '' } = evt);
```

Comment:
Should we type the env id & status to be `string?`, to make sure nobody is passing null, in which case this wouldn't be equivalent?

Reply:
`evt` is `any` so it goes much further than that, it would be nice to have it better type checked but it's a much bigger lift

Reply:
From outside discussion the underlying concern is about the default operator only applying for `undefined` and not `null`, rather than about typings specifically